Source/RestKit/RestRequest.swift

```diff
@@ -303,7 +303,36 @@ public struct RestRequest {
             completionHandler(dataResponse)
         }
     }
-    
+
+    public func responseVoid(
+        responseToError: ((HTTPURLResponse?, Data?) -> Error?)? = nil,
+        completionHandler: @escaping (RestResponse<Void>) -> Void)
+    {
+        response() { data, response, error in
+
+            if let responseToError = responseToError,
+                let error = responseToError(response, data) {
```

Comment:
I personally have a distaste for Xcode's recommended style with multiple `if let` statements. When I first read this, my eye didn't catch the trailing comma on line 313. So I thought, "This looks like a bug. I don't think we want to execute this block _every_ time the `responseToError` function is not `nil`. Don't we want to try to parse an error first?"

Unfortunately, I haven't found a great alternative for styling multiple `if let` statements. Since this line is (relatively) short, I think the best choice in this case would be to put both conditions on the same line:

```
if let responseToError = responseToError, let error = responseToError(response, data) {
    ....
}
```

I personally find that easier to read and understand at a glance. Otherwise the second condition blends in with the code block itself.